.pre-commit-config.yaml

```diff
@@ -21,25 +21,25 @@ repos:
     -   id: debug-statements
         language_version: python3
 -   repo: https://github.com/ambv/black
-    rev: 20.8b1
+    rev: 21.4b0
     hooks:
     -   id: black
         language_version: python3
-        args: ["--target-version", "py36"]
+        args: ["--target-version", "py37"]
 -   repo: https://github.com/pycqa/flake8
-    rev: 3.8.4
+    rev: 3.9.1
     hooks:
     -   id: flake8
         language_version: python3
         args: ['--config=setup.cfg']
 -   repo: https://github.com/timothycrosley/isort
-    rev: 5.7.0
+    rev: 5.8.0
     hooks:
     -   id: isort
         language_version: python3
-        args: ['--profile', 'black', '--filter-files', '--py', '36']
+        args: ['--profile', 'black', '--filter-files', '--py', '37']
```

Comment:
I would suggest we get rid of the `--py` option, I don't see the point of keeping it.

Reply:
Okay. The only change dropping it would do is we _might_ see some changes happen without our knowledge if we use some new library in the standard lib that's been backported to an older python. This is a really, really specific case, so that's fine.